nntrainer/include/neuralnet.h

```diff
@@ -129,6 +130,8 @@ class NeuralNetwork : public ml::train::Model {
    */
   int loadFromConfig(std::string config);
 
+  int compile();
```

Comment:
I think it might be good to expose this with c++ API?

Reply:
Yes. I will. Thanks.